src/main/java/mate/jdbc/service/CarServiceImpl.java

```diff
@@ -19,7 +19,7 @@ public Car create(Car car) {
 
     @Override
     public Car get(Long id) {
-        return carDao.get(id);
+        return carDao.get(id).get();
```

Comment:
return carDao.get(id).orElseThrow(() ->
                new RuntimeException("There is no car with id " + id));